### PR TITLE
UICONSET-111 Protect create, edit, and delete actions by selected members' permissions.

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UICONSET-111

Don't display controlled vocabulary actions if the user doesn't have appropriate permissions in the selected tenants or in the consortium manager (for shared records);
## Approach

- Use publish coordinator to get current user's permissions in all selected tenants. The value will wi invalidated on member selection;
- Provide a tool for consortium manager context to check required permissions;'
- Update consortia controlled vocabulary settings to specify permissions for each operation;
- Add unit tests;

## Screencast
chrome_FUwXSSmSxO

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by usavkov-epam** [comment]
@folio-org/acquisitions-ui please take a look at the PR. Thanks in advance!

File: src/constants.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const METADATA_PROVIDER_API = 'metadata-provider';
 export const MODES_OF_ISSUANCE_API = 'modes-of-issuance';
 export const NATURE_OF_CONTENT_TERMS_API = 'nature-of-content-terms';
 export const PATRON_GROUPS_API = 'groups';
+export const PERMISSIONS_USERS_API = 'perms/users';
```
Comment (alisher-epam) [inline src/constants.js:34]:
```suggestion
export const PERMISSION_USERS_API = 'perms/users';
```

Comment (alisher-epam) [reply src/constants.js:34]: It is easier to pronounce and matches other constant names as well

File: src/hooks/useUserTenantsPermissions/useUserTenantsPermissions.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { useQuery } from 'react-query';
+
+import { useNamespace } from '@folio/stripes/core';
+
+import { usePublishCoordinator } from '../usePublishCoordinator';
+import { PERMISSIONS_USERS_API } from '../../constants';
```
Comment (alisher-epam) [inline src/hooks/useUserTenantsPermissions/useUserTenantsPermissions.js:6]: Swap the order of import lines please

**Comment by alisher-epam** [comment]
@usavkov-epam Except above comment, the rest looks good to me.

File: src/hooks/useUserTenantsPermissions/useUserTenantsPermissions.js (hunk 2 above)
Comment (usavkov-epam) [reply src/hooks/useUserTenantsPermissions/useUserTenantsPermissions.js]: updated

File: src/constants.js (hunk 1 above)
Comment (usavkov-epam) [reply src/constants.js]: updated

**Review by alisher-epam — APPROVED** [review]
LGTM

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**